javascript/javascript3/week1/Weather_app/secret.js

```diff
@@ -0,0 +1 @@
+const appid = "bf2cb42517c3fb29daf26002f9312b40";
```

Comment:
This is clearly not in your .gitignore file

Reply:
Yeah... I added the file first and after that made changes in gitignore. Is there any way to add it now.

Reply:
Delete it from your git repository since .gitignore files aren't added retroactively. After that you can include it back to your local repository. That being said, since this repo is public this key is already available online and there's nothing you can do about it. If this was a security critical application, I would deactivate this API key.